packages/trader/src/Types/common-prop.type.ts

```diff
@@ -14,4 +14,9 @@ export type TProposalTypeInfo = {
     stake: string;
 };
 
+export type TError = {
```

Comment:
where is this type is used ? )

Reply:
i saw it and copied it from ur one of your PRs